src/main/scala/com/softwaremill/mqperf/MessagesPool.scala

```diff
@@ -0,0 +1,22 @@
+package com.softwaremill.mqperf
+
+import scala.util.Random
+
+object MessagesPool {
+  final val DEFAULT_MESSAGES_POOL_SIZE = 10000
```

Comment:
in Scala the convention is to write constants using normal camel case with a capital letter, e.g. `DefaultMessagesPoolSize`

Reply:
Thanks, corrected.